### PR TITLE
[4.12] OCPBUGS-6789: honor feature gates during bootstrapping

### PR DESCRIPTION
**Pull request by stlaz**
backport of https://github.com/openshift/cluster-kube-controller-manager-operator/pull/686

/cc @deads2k @ibihim

### CODE REVIEW

**Comment by deads2k** [comment]
/approve
/assign @ibihim 

for lgtm

File: bindata/bootkube/config/bootstrap-cluster-policy-controller-config-overrides.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@ servingInfo:
   clientCA: /etc/kubernetes/secrets/kube-control-plane-ca-bundle.crt
   certFile: /etc/kubernetes/secrets/kube-control-plane-kube-controller-manager-client.crt
   keyFile: /etc/kubernetes/secrets/kube-control-plane-kube-controller-manager-client.key
+featureGates: {{range .FeatureGates}}
+    - {{.}}{{end}}
```
Comment (ibihim) [inline bindata/bootkube/config/bootstrap-cluster-policy-controller-config-overrides.yaml:8]: Do we want to have a proper file ending? Or do we ignore it on a backport?

File: bindata/bootkube/config/bootstrap-config-overrides.yaml [hunk 2]
```diff
@@ -25,3 +25,5 @@ extendedArguments:
   - ""
   pv-recycler-pod-template-filepath-hostpath:
   - ""
+  feature-gates: {{range .FeatureGates}}
+    - {{.}}{{end}}
```
Comment (ibihim) [inline bindata/bootkube/config/bootstrap-config-overrides.yaml:29]: Do we want to have a proper file ending? Or do we ignore it on a backport?

**Review by ibihim — APPROVED** [review]
/lgtm

**Pull request merged**